functional-tests/devscripts/run_tests.sh

```diff
@@ -10,7 +10,7 @@ function printHelp {
 	YELLOW="\\033[93;1m"
 	WHITE="\\033[0;1m"
 	GREEN="\\033[32;1m"
-	NC='\033[0m' # No Color
+	NC='\\033[0m' # No Color
```

Comment:
Why there is only one ```\``` and others stay?

Reply:
? the `\` backwards slash was missing. others use doubleslash correctly

Reply:
what's more intriguing to me is why `NC` uses `'` instead of `"`

Reply:
Yes, maybe you can try it and fix that. I think it will not take a lot of time.